scripts/mainsite.js

```diff
@@ -0,0 +1,213 @@
+var lastStreamer, existingIframe;
+
+window.addEventListener("message", (event) => {

```

Comment:
How often does this occur? Will this script have the potential to lag out? For `embAdbQuality`, is that something that twitch has set that we are trying to inject something to so twitch reads a different value? Or is it just something we are storing to refer to later, in which case wouldn't a variable we define be better? 

I do see down below we use this `localStorage` but I can't help but think that we can optimize it just by using a variable.